Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -192,7 +216,6 @@ void SetupIDs()
         {
             if (ThisIsAPrefab())
             {
-                ForceSceneId(0);
```

Comment:
why is this removed?
sceneIds should be 0 for prefabs

Reply:
This is the big change, sceneId's are no longer zero for prefabs as their identity is set at edit time.  Then during the `NetworkScenePostProcess` the offset is added for additive loading.

Reply:
if I understand correctly, there is a chance that at runtime, a prefab will be instantiated into the scene which has a != 0 sceneId even though it wasn't in the scene since the beginning?

Reply:
That's correct, there are three possible scenarios here:

1. User drags a prefab into the scene, it registers itself with the `NetworkIdentityManager` and if the sceneId is already in use, it will be assigned a new unique ID instantly.  Even if the prefab sceneId != 0, it will always be given a unique ID.

2. If a user duplicates an existing prefab in the scene via copy paste, the same method above will run and guarantee a unique scene ID upon creation.